concepts/05 UI Components/TreeList/00 Getting Started with TreeList/20 Sort Data.md

```diff
@@ -0,0 +1,103 @@
+The **sorting**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/sorting/#mode) property specifies whether users can sort records by single or multiple columns. This tutorial uses the default sorting mode - single. 
+
+You can also set a column's [sortOrder](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#sortOrder) and [sortIndex](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#sortIndex) properties to specify the initial sorting settings. **sortIndex** applies only in multiple sorting mode.
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#treeList").dxTreeList({
+            // ...
+            columns: [{
+                dataField: "Position",
+                sortOrder: "asc",
+            },
+            // ...
+            ],
+            // sorting: { mode: "single" },
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-tree-list ... >
+        <!-- ... -->
+        <dxi-column
+            dataField="Position"
+            sortOrder="asc">
```

Comment:
In the source code, the State column is used for sorting: https://github.com/DevExpress-Examples/getting-started-with-treelist/blob/20.2.3%2B/angular/src/app/app.component.html#L33